CorsixTH/Lua/languages/german.lua

```diff
@@ -286,67 +319,80 @@ adviser = {
 dynamic_info.patient.actions.no_gp_available = "Wartet darauf, dass Sie eine Allgemeinmedizin bauen"
 dynamic_info.staff.actions.heading_for = "Geht zu: %s"
 dynamic_info.staff.actions.fired = "Gefeuert"
+dynamic_info.staff.actions.vaccine = "Impft einen Patienten"
+dynamic_info.patient.actions.epidemic_vaccinated = "Ich bin nicht mehr ansteckend"
```

Comment:
This is out of scope but a comment for a future PR -- these should probably be changed to be nested.

Reply:
I would have done that, but I tried to match english.lua to facilitate side-by-side diffing, so what wasn't nested there isn't nested here.